### PR TITLE
fix(postgres): use schema set in sequelize config by default

### PR DESCRIPTION
**Pull request by nholmes3**
### Pull Request Checklist

_Please make sure to review and check all of these items:_

- [x] Have you added new tests to prevent regressions?
- [x] Does `yarn test` or `yarn test-DIALECT` pass with this change (including linting)?
- [x] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [x] Did you update the typescript typings accordingly (if applicable)?
- [x] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
- [x] Did you follow the commit message conventions explained in [CONTRIBUTING.md](https://github.com/sequelize/sequelize/blob/main/CONTRIBUTING.md)?

### Description Of Change

This has already been merged into `main` https://github.com/sequelize/sequelize/pull/14634

This is an attempt at fixing a bug where the Postgres Query Generator does not respect a custom schema name if it is provided.

https://github.com/sequelize/cli/issues/284#issuecomment-573351082 on https://github.com/sequelize/cli/issues/284 describes the problem in detail.

### CODE REVIEW

**Comment by nholmes3** [comment]
@WikiRik @ephys Is it possible to merge these changes here too so that we can get this out in v6? Thanks!

File: test/unit/sql/remove-column.test.js [hunk 1]
```diff
@@ -1,28 +1,50 @@
...
+describe(Support.getTestDialectTeaser('SQL'), () => {
+  describe('removeColumn', () => {
+    if (current.dialect.name !== 'sqlite') {
       it('schema', () => {
         expectsql(sql.removeColumnQuery({
           schema: 'archive',
-          tableName: 'user'
+          tableName: 'user',
         }, 'email'), {
+          ibmi: 'ALTER TABLE "archive"."user" DROP COLUMN "email"',
```
Comment (WikiRik) [inline test/unit/sql/remove-column.test.js:24]:
```suggestion
```
ibmi is v7 only

File: test/unit/sql/remove-column.test.js [hunk 2]
```diff
@@ -1,28 +1,50 @@
...
+          snowflake: 'ALTER TABLE "archive"."user" DROP "email";',
         });
       });
+    }
+
+    it('defaults the schema to the one set in the Sequelize options', () => {
+      expectsql(customSql.removeColumnQuery({
+        tableName: 'user',
+      }, 'email'), {
+        ibmi: 'ALTER TABLE "user" DROP COLUMN "email"',
```
Comment (WikiRik) [inline test/unit/sql/remove-column.test.js:39]:
```suggestion
```

File: test/unit/sql/add-column.test.js [hunk 3]
```diff
@@ -1,68 +1,88 @@
...
+        allowNull: false,
+      })), {
+        mariadb: 'ALTER TABLE `Users` ADD `level_id` FLOAT NOT NULL;',
+        mysql: 'ALTER TABLE `Users` ADD `level_id` FLOAT NOT NULL;',
+        postgres: 'ALTER TABLE "custom"."Users" ADD COLUMN "level_id" FLOAT NOT NULL;',
+        sqlite: 'ALTER TABLE `Users` ADD `level_id` FLOAT NOT NULL;',
+        mssql: 'ALTER TABLE [Users] ADD [level_id] FLOAT NOT NULL;',
+        db2: 'ALTER TABLE "Users" ADD "level_id" FLOAT NOT NULL;',
+        snowflake: 'ALTER TABLE "Users" ADD "level_id" FLOAT NOT NULL;',
+        ibmi: 'ALTER TABLE "Users" ADD "level_id" FLOAT NOT NULL',
```
Comment (WikiRik) [inline test/unit/sql/add-column.test.js:84]:
```suggestion
```

**Review by WikiRik — COMMENTED** [review]
Bigger diff than needed, but looks good. Three small things to fix and I think we can approve

File: test/unit/sql/remove-column.test.js (hunk 1 above)
Comment (nholmes3) [reply test/unit/sql/remove-column.test.js]: removed

File: test/unit/sql/remove-column.test.js (hunk 2 above)
Comment (nholmes3) [reply test/unit/sql/remove-column.test.js]: removed

File: test/unit/sql/add-column.test.js (hunk 3 above)
Comment (nholmes3) [reply test/unit/sql/add-column.test.js]: removed

**Review by WikiRik — APPROVED** [review]

**Comment by nholmes3** [comment]
@ephys any movement here? We would love to get this out in v6 as soon as possible. Thank you for your help :)

**Review by ephys — APPROVED** [review]
This should really be fixed in all dialects but I'm ok with starting with just PG

**Pull request merged**